apps/backend/src/mutations/FapMutations.ts

```diff
@@ -725,6 +725,25 @@ export default class FapMutations {
     agent: UserWithRole | null,
     args: ReorderFapMeetingDecisionProposalsInput
   ): Promise<FapMeetingDecision | Rejection> {
+    const [{ instrumentId, fapId, proposalPk }] = args.proposals;
```

Comment:
It looks like args.proposals is a list of many proposals, but that we're only using a single proposal's details in the checks. I'd have thought we'd need to use over all of them?

Reply:
I understand your concern but the information that I am pulling is the same for all proposals. I get one of the `proposalPk` to be able to get the `callId`. All of the proposals here are on the same call, instrument and fap. So I use the first one to just extract some information that is needed for later checks.